experiments/ClimaCore/sea_breeze/atmos_rhs.jl

```diff
@@ -162,26 +207,26 @@ function atm_rhs!(dY, Y, params, t)
     @. dYc.ρuₕ = -κ₄ * hwdiv(Yc.ρ * hgrad(dYc.ρuₕ))
     @. dρw = -κ₄ * hwdiv(Yfρ * hgrad(dρw))
 
-    # density
+    ## density
     @. dYc.ρ = -∂(ρw)
     @. dYc.ρ -= hdiv(Yc.ρuₕ)
 
-    # potential temperature
+    ## potential temperature
     @. dYc.ρθ += -(∇_z_ρθ(ρw * If(Yc.ρθ / Yc.ρ)))
     @. dYc.ρθ -= hdiv(uₕ * Yc.ρθ)
 
-    # horizontal momentum
+    ## horizontal momentum
     Ih = Ref(Geometry.Axis2Tensor((Geometry.UAxis(), Geometry.UAxis()), @SMatrix [1.0]))
     @. dYc.ρuₕ += -uvdivf2c(ρw ⊗ If(uₕ))
     @. dYc.ρuₕ -= hdiv(Yc.ρuₕ ⊗ uₕ + p * Ih)
 
-    # vertical momentum
+    ## vertical momentum
     @. dρw +=
         B(Geometry.transform(Geometry.WAxis(), -(∂f(p)) - If(Yc.ρ) * ∂f(Φ(center_coords.z))) - vvdivc2f(Ic(ρw ⊗ w)))
     uₕf = @. If(Yc.ρuₕ / Yc.ρ) # requires boundary conditions
     @. dρw -= hdiv(uₕf ⊗ ρw)
 
-    ### UPWIND FLUX CORRECTION
+    ## UPWIND FLUX CORRECTION
```

Comment:
more info?

Reply:
@akshaysridhar assigning this one to you ;)